src/server/migrations/0.8.0-1.0.0/index.js

```diff
@@ -10,6 +10,6 @@ module.exports = {
 	toVersion: '1.0.0',
 	up: async db => {
 		await db.none(sqlFile('../migrations/0.8.0-1.0.0/sql/create_cik_table.sql'));
-		await db.none(sqlFile('../migrations/0.8.0-1.0.0/sql/create_compressed_reading_views.sql'));
+		await db.none(sqlFile('../migrations/0.8.0-1.0.0/sql/create_reading_views.sql'));
```

Comment:
I think we also need to call `drop_old_functions.sql` and `drop_old_views.sql` here. I don't have good knowledge of creating migrations so please correct me if I'm wrong. Thank you.